ts/commands/ElementCommands.ts

```diff
@@ -63,6 +63,42 @@ class ElementCommands {
         const config = configService.mergeConfig(options);
         elementService.stopMaintenanceMode(args.id, config, logger);
       });
+
+    (caporal as any)
+      .command('image custom delete', 'Delete a custom image')
```

Comment:
Now that these are not prefixed by element can you move them to a new file `ImageCommands.ts`.